src/rabbit_mgmt_db_cache.erl

```diff
@@ -92,7 +92,7 @@ start_link(Key) ->
 %%%===================================================================
 
 init([]) ->
-    Mult = application:get_env(rabbitmg_management, management_db_cache_multiplier,
+    Mult = application:get_env(rabbitmq_management, management_db_cache_multiplier,
```

Comment:
Not sure about this one

Reply:
That's a legitimate issue fixed (a configuration parameter would never apply because of this typo) 👍